DSE_6.8_Release_Notes.md

```diff
@@ -2,6 +2,48 @@
 DSE 6.8.x is compatible with Apache Cassandra&trade; 3.11 and adds additional production-certified changes, if any.
 Components that are indicated with an asterisk (&ast;) (if any) are known to be updated since the prior patch version.
 
+# Release notes for 6.8.37
+10 July 2023
+
+## Components versions for DSE 6.8.37
+ * Apache Solr™ 6.0.1.4.2959
+ * Apache Spark™ 2.4.0.28
+ * Apache TinkerPop™ 3.4.14-20230523-37856751
+ * Apache Tomcat® 8.5.89
+ * DSE Java Driver 1.10.0-dse-20220616 (DSE *internal-only* version)
+ * Netty 4.1.86.1.dse
+ * Spark JobServer 0.8.0.54
+
+**NOTE**: above-listed DSE Java Driver is an _internal-version_ only.
+If you're developing applications, please refer to the [Java Driver documentation](https://docs.datastax.com/en/driver-matrix/doc/java-drivers.html) to choose an appropriate version.
+
+## 6.8.37 DSE Core
+* Allowed to encrypt `kmip_host` passwords in the `dse.yaml` file. New optional parameter `password_encryption_key_name` was added to the `kmip_hosts` items. DSP-23326 enabled password encryption before placing them in `dse.yaml`. (DSP-23313)
+* Fixed the issue with dropping the UDT column, which potentially leads to a problem accessing data from the affected table after the node is restarted. (DSP-23337)
+* RSA Certificates for SSL on Java 11 require larger keypairs compared to Java 8. Testing on Java 11 uses 1024-bit keys instead of 512-bit keys. (DSP-23374)
```

Comment:
Same as https://github.com/datastax/release-notes/pull/104#discussion_r1257877073

Reply:
Good catch. Jira ticket has `Doc Impact` set to both: "No Documentation Needed" and "Release Notes". I'll add some logic in the `release-notes-generator` to catch these cases! :)

Reply:
Having sth like: "if contains "no doc needed then skip RNs" would be useful.

Reply:
What about:
```
2023-07-10 10:12:44,827 : WARNING : [DSP-23374] "Doc Impact" has incoherent tags ['Release Notes', 'No Documentation Needed'] - release notes will be generated
```
?

If there's no "RN: " there will be a second warning saying that it will skip it because they're missing. In this case, there was a "RN: " that's why I didn't catch this!

Reply:
I rather assume that we'll be generating the release notes and issue a warning to double check than skipping the release notes generation and then having to regenerate again (removing an issue being cheaper than adding one).

Reply:
Right, that is better, `warn` motivates the user to verify what's going on.